SofaKernel/modules/SofaCore/src/sofa/core/loader/MeshLoader.cpp

```diff
@@ -83,18 +83,6 @@ MeshLoader::MeshLoader() : BaseLoader()
     d_transformation.setAutoLink(false);
     d_transformation.setDirtyValue();
 
-    d_positions.setPersistent(false);
```

Comment:
This is breaking, but this is really the way to go..anyway un the current state of the sofa code, saving is just un-unsable (and unused).

Reply:
Good point. I set the breaking label on the PR